trakt/errors.py

```diff
@@ -143,6 +144,21 @@ def details(self):
             return None
 
 
+class AccountLimitExceeded(RateLimitException):
+    """TraktException type to be raised when a 420 return code is received"""
+    http_code = 420
+    message = 'Account Limit Exceeded - list count, item count, etc'
+
+    @property
+    def account_limit(self):
+        """Get the account limit details from response headers.
+
+        Returns:
+            str|None: The value of x-account-limit header or None if not present
+        """
+        return self.response.headers.get("x-account-limit", None)
```

Comment:
I did tests and I do not get this `x-account-limit` header in 420 response.

I only get `x-ratelimit` with :
```json
{"name":"AUTHED_API_POST_LIMIT","period":1,"limit":1,"remaining":0,"until":"2025-01-28T14:56:57Z"}
```

Tested with [Add to collection](https://trakt.docs.apiary.io/#reference/sync/add-to-collection/add-items-to-collection) url.